smartsim/_core/entrypoints/redis.py

```diff
@@ -57,8 +57,15 @@ def main(network_interface: str, command: List[str]):
 
     try:
 
-        ip_address = current_ip(network_interface)
-        cmd = command + [f"--bind {ip_address}"]
+        ip_addresses = [
+            current_ip(net_if) for net_if in network_interface.split(",")
+        ]
+        ip_address = ip_addresses[0]
+        cmd = command + ["--bind " + " ".join(ip_addresses)]
```

Comment:
Now purely stylistic: Can we also make this an f-str to be consistent with the colo script?

Reply:
Yeah, it always bothered me.